supabase/functions/check_contact_request/index.ts

```diff
@@ -1,10 +1,16 @@
 import { createClient } from "https://esm.sh/@supabase/supabase-js@2";
 import { checkIfContactRequestIsAllowed } from "../_shared/checks.ts";
 import { corsHeaders } from "../_shared/cors.ts";
```

Comment:
Why allow PUT, DELETE and POST in. shared headers if you only do GET requests?

Reply:
I have edge functions which are called with GET and some are called with POST. I also need OPTIONS for the actual preflight request. I removed PUT and DELETE. https://github.com/technologiestiftung/giessdenkiez-de-postgres-api/pull/269/commits/e1985031c8c36aa9e9538371f592ef6f5f2e9835